src/AutoCrudAdmin/Controllers/AutoCrudAdminController.cs

```diff
@@ -742,7 +742,8 @@ protected string GetComplexFormControlNameFor<T>()
     private static IGridColumnsOf<TEntity> GenerateColumnConfiguration<TProperty>(
         IGridColumnsOf<TEntity> columns,
         PropertyInfo property,
-        int? columnStringMaxLength)
+        int columnStringMaxLength,
```

Comment:
Who passes this? Is it passed as `int` there as well?

Reply:
No. I only modified this parameter. Otherwise, I have to change 
` protected virtual int? ColumnStringMaxLength => Grid.DefaultColumnStringMaxLength;` to an integer which may be unwanted.

Reply:
Why might be unwanted?